packages/dashboard/src/components/sidePanel/utils/index.ts

```diff
@@ -46,4 +46,5 @@ export const useInput: <T>(path: string, validator?: (newValue: T) => boolean) =
 
 // this helps Typescript finding correct types given attribute names
```

Comment:
Is there some implementation we're missing here with generics? 

```js
export const useInput<T> = useInput<T>;
```

Reply:
I am not sure but worth tackling this at some point definitely. Out of scope for this PR though.